### PR TITLE
Change minimum garbage_collection TTL to 1 day from 7

### PR DESCRIPTION
**Pull request by ericreeves**
A very simple change to modify the minimum TTL on the garbage collector, pulling it back to 1 day from 7.

It still seems quite reasonable to ensure the user doesn't specify "0".

### CODE REVIEW

**Review by armab — CHANGES_REQUESTED** [review]
Thanks for finding this setting and for the contribution @ericreeves ! :+1: 
We'll just need a CHANGELOG record for this and CLA signed based on Github bot message above.

Besides that, all good.

As for the current 3.5.0 we got code freeze already from the release team, the change will be included in the next release.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -263,6 +263,10 @@ Changed
 
 * Update orquesta to v1.4.0.
 
+* Reduced minimum TTL on garbage collection for action executions and trigger instances from 7 days to 1 day.
```
Comment (armab) [inline CHANGELOG.rst:266]:
```suggestion
* Reduced minimum TTL on garbage collection for action executions and trigger instances from 7 days to 1 day. #5287
```

**Review by armab — APPROVED** [review]
:+1:

**Review by Kami — APPROVED** [review]
Yeah that change seems reasonable to me :+1:

**Pull request merged**